test/transforms.jl

```diff
@@ -214,7 +216,8 @@
       p₂ = scatter(n.x, n.y, label="Center")
       p = plot(p₁, p₂, layout=(1,2))
 
-      @test_reference joinpath(datadir,  "center.png") p
+      @test_reference joinpath(datadir, "center.png") p
+      # savefig(p, joinpath(datadir,  "center.png"))
```

Comment:
```suggestion
```

Reply:
I removed the comments of savefig function.